src/DatabaseController.js

```diff
@@ -24,4 +24,6 @@ function createUserAccount(name, password, date, email){
         });
       });
 }
+createUserAccount("bajs", "kiss", "carlsson", true)
```

Comment:
Is this a testing function call?

Reply:
Yes, this should be removed, was only for testing purposes